src/components/Login/Login.js

```diff
@@ -0,0 +1,89 @@
+import React, {useEffect, useState} from 'react';
+import './Login.css';
+
+export const Login = ({logIn, isLoggedIn}) => {
+  const [email, setEmail] = useState("")
+  const [password, setPassword] = useState("")
+  const [error, setError] = useState("")
+  const [disabled, setDisabled] = useState(true)
+
+  const handleUserChange = e => {
+    setEmail(e.target.value)
+    if (password && email) {
+      setDisabled(false)
+    }
+  }
+
+  const handlePassChange = e => {
+    setPassword(e.target.value)
+    if (password && email) {
+      setDisabled(false)
+    }
+  } 
+
+  const clearInputs = () => {
+    setEmail("")
+    setPassword("")
+  }
+
+  const submitCredentials = async e => {
+    e.preventDefault()
+    const creds =   { 
+      email: email.trim(),
+      password: password.trim()
+    }
+
+    setError("")
+    try {
+      await logIn(creds)
+      clearInputs()
+    } catch {
+      setError("Mismatch")
```

Comment:
Should we make this a more specific message?

Reply:
We can! But the error message shows up the same no matter what because of the JSX in the Login component.

Reply:
okay thats fine!